lms/djangoapps/instructor/views/api.py

```diff
@@ -2789,11 +2785,18 @@ def add_certificate_exception(course_key, student, certificate_exception):
         }
     )
 
+    generated_certificate = GeneratedCertificate.objects.filter(
+        user=student,
+        course_id=course_key,
+        status=CertificateStatuses.downloadable
```

Comment:
Wherever syntactically valid, dicts/lists/sets/kwargs should always contain trailing commas. It provides a cleaner `diff` in the event of future additions/subtractions, which reduces the likelihood of conflicts for those running on a forked repository.